src/lib/libfs.js

```diff
@@ -487,6 +487,13 @@ FS.staticInit();
       stream.stream_ops?.dup?.(stream);
       return stream;
     },
+    doSetAttr(stream, node, attr) {
```

Comment:
@sbc100 this is used 3 times so I think it's reasonable to keep as a separate function?